MODULE.md

```diff
@@ -0,0 +1,262 @@
+
+- Automatic Inflation: When enabled, tokens are minted every block, aiming for a predetermined total over a year.
+
+#### Commands
+
+##### Update Parameters (update-params):
+
+- Syntax: `manifestd tx manifest update-params [address:percent_share] [inflation_on_off] [annual_total_mint]`
+
+  - Parameters:
+    - `address:percent_share`: Specifies the wallet address and its share of the total rewards (to the ninth exponent).
+    - `inflation_on_off`: A boolean value (true or false) to toggle automatic inflation.
+    - `annual_total_mint`: The total amount of tokens to be minted annually.
+
+  **Example:** `manifestd tx manifest update-params manifest1hj5fveer5cjtn4wd6wstzugjfdxzl0xp8ws9ct:100_000_000 false 500000000umfx`
+
+##### Stakeholder Payout (stakeholder-payout):
+
+- Syntax: `manifestd tx manifest stakeholder-payout [amount]`
+
+  - Parameters:
+
+    - `amount`: The amount of tokens to mint from the yearly allocated inflation.
+
+    This command will fail if automatic inflation is enabled.
+
+  **Example:** `manifestd tx manifest stakeholder-payout 777umfx`
+
+## Proof of Authority Module
+
+The PoA module is responsible for handling admin actions like adding and removing other administrators, setting the staking parameters of the chain, controlling voting power, and allowing/blocking validators. Below is a structured breakdown of its components and functionalities:
+
+### Module Functionality
+
+The PoA admin has several capabilities for managing the chain and its validators:
+
+#### Validator Management:
+
+- Remove validators from the active set.
+- Remove validators pending addition to the active set.
+- Specify the voting power for each validator.
+- Approve the addition of new validators.
+
+#### Staking Parameters Update:
+
+- The PoA admin can update various staking parameters, including:
+  - `unbondingTime`: The time period for unbonding tokens.
+  - `maxVals`: The maximum number of validators.
+  - `maxEntries`: The maximum number of entries.
+  - `historicalEntries`: The number of historical entries to store.
+  - `bondDenom`: The denomination for bonding and staking.
+  - `minCommissionRate`: The minimum commission rate for validators.
+
+#### Administrative Rights:
+
+- Assign or revoke administrative privileges.
+- Determine if validators have the ability to self-revoke.
+
+#### Commands
+
+##### Update Parameters (update-params):
+
+- Syntax: `manifestd tx poa update-params [admin1,admin2,admin3,...] [allow-validator-self-exit-bool]`
+
+  - Parameters:
+    - `admin1,admin2,admin3,...`: A list of admin addresses which can be multisig addresses.
+    - `allow-validator-self-exit-bool`: A boolean value (true or false) to allow validators to self-exit.
+
+  **Example:** `manifestd tx poa update-params manifest1hj5fveer5cjtn4wd6wstzugjfdxzl0xp8ws9ct,manifest1hj5fveer5cjtn4wd6wstzugjfdxzl0xp8ws9ct false`
+
+##### Update Staking Parameters (update-staking-params):
```

Comment:
I assume this is NOT used when automatic inflation is disabled as there is no staking? If so, please add a mention.

Reply:
Your assumption is incorrect. Staking parameters have no correlation to inflation.

Reply:
Please explain further!

Reply:
Thank you for the clarifications!